client/src/utils/API.js

```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
+ const baseUrl = `http://localhost:8000`
+
+export default {
+  getUser: (config) => {
+      return  axios.get(baseUrl+"/api/auth/user", config);
+    },
```

Comment:
when you're returning a single value from an arrow function, do it same-line.

Reply:
Corrected the entire file. I had written it this way originally and wasn't sure if the one line  arrow functions looked to cluttered. I prefer the one line.